README.md

```diff
@@ -17,6 +17,28 @@ Use the `-log.level` flag. Run the agent with the `-help` flag to see the possib
 | `info`       | 0 (`-v` not set) |
 | `debug`      | 3 (`-vvv`)       |
 
+## Environment Variables and Flags
+
+If using commandline flags is difficult to use in your environment, you can define environment variables to be consumed by the PDC Agent instead. *Commandline flags always take precedence.*
+
+The following mappings are currently provided:
+
+| Flag                        | Environment Variable                  |
+|-----------------------------|---------------------------------------|
+| `-token`                    | `GCLOUD_PDC_SIGNING_TOKEN`            |
+| `-cluster`                  | `GCLOUD_PDC_CLUSTER`                  |
+| `-gcloud-hosted-grafana-id` | `GCLOUD_PDC_GCLOUD_HOSTED_GRAFANA_ID` |
```

Comment:
```suggestion
| `-gcloud-hosted-grafana-id` | `GCLOUD_HOSTED_GRAFANA_ID` |
```

Reply:
Do we want variables prefixed with `GCLOUD_PDC` or no? This and the above comment are contradictory

Reply:
> Do we want variables prefixed with GCLOUD_PDC or no?

Yes please :)

Reply:
Addressed in 111c4c751cf015056664b49076a43745df9abd81